pda_aq_programmer.c

```diff
@@ -858,14 +852,46 @@ bool set_PDA_aqualink_freezeprotect_setpoint(struct aqualinkdata *aq_data, int v
   } else if (! goto_pda_menu(aq_data, PM_FREEZE_PROTECT)) {
     logMessage(LOG_ERR, "Error finding freeze protect setpoints menu\n");
     return false;
-  } else if (! set_PDA_numeric_field_value(aq_data, val, &aq_data->frz_protect_set_point, NULL, 1)) {
+  } else if (! set_PDA_numeric_field_value(aq_data, val, aq_data->frz_protect_set_point, NULL, 1)) {
     logMessage(LOG_ERR, "Error failed to set freeze protect temp value\n");
     return false;
   } else {
       return waitForPDAnextMenu(aq_data);
   }
 }
 
+bool set_PDA_aqualink_time(struct aqualinkdata *aq_data) {
+  if (! goto_pda_menu(aq_data, PM_SET_TIME)) {
```

Comment:
going to SET_TIME menu will update aq_data->tm. And every time the menu is updated the tm value is also updated